src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -85,6 +88,15 @@ internal VisualStudioProject(VisualStudioWorkspaceImpl workspace, HostDiagnostic
             _displayName = projectUniqueName;
             _projectUniqueName = projectUniqueName;
 
+            if (language == LanguageNames.CSharp)
```

Comment:
There really shouldn't be special language checks like this. I would assume we *also* want it null for C# as a *default* as "I don't know". There are project types that will create projects like this that won't ever initialize this value, but you don't want to act upon it.

Reply:
That's exactly why this code wasn't in the first iteration, I had a brain fart apparently... thanks for pointing out